src/ethereum_test_types/eof/v1/__init__.py

```diff
@@ -200,12 +200,12 @@ def type_definition(self) -> bytes:
                     auto_code_outputs,
                 )
 
-        assert max_stack_height >= code_inputs, "incorrect max_stack_height value"
-
+        max_stack_increase = max_stack_height - code_inputs
+        assert max_stack_increase >= 0, "incorrect max stack height value"
```

Comment:
funny, when I was trying the big branch with all eof-devnet-1, this assertion was failing for me quite a lot. Is it filling all the tests now?

Reply:
No failures for me.

Reply:
ok, maybe something broken on my end